superset-frontend/packages/superset-ui-chart-controls/src/components/Tooltip.tsx

```diff
@@ -46,9 +46,15 @@ export const Tooltip = ({ overlayStyle, color, ...props }: TooltipProps) => {
         overlayStyle={{
           fontSize: theme.typography.sizes.s,
           lineHeight: '1.6',
+          maxWidth: 250,
+          minWidth: 120,
```

Comment:
Might as well make this gridUnit based, too.... it's a nice round number at least :D